content/committees/specification/tckprocess/index.md

```diff
@@ -46,16 +46,24 @@ namespace format of `ee.jakarta.tck.<spec-name>` is recommended.
 
-The individual specification TCK must contain Type 2 or Type 3 tests while Type 1 tests are optional. A profile-ready mechanism must be provided for running the required TCK tests (types 2 and 3) in the profiles they are included in: platform, web profile or core profile. Type 1 tests must be excluded and not required to be executed.
+- Type 1: Behavior defined by the specification that is _only_ applicable when the implementation is running outside of a platform (or profile), such as on Java SE, to test behavior defined by the specification for that mode of execution.
+- Type 2: Behavior defined by the specification that is _only_ applicable when the implementation is running in a platform (or profile).
+- Type 3: Behavior defined by the specification that is applicable _both_ when the implementation is running outside of a platform (or profile) _and_ when it is running in a platform (or profile).
 
-The individual specification TCK must provide a mechanism for running the required TCK tests (types 1 and 3) outside the context of a platform or profile. Type 2 tests must be excluded and not run.
+Not all individual specifications will have Type 1 or Type 2 requirements. The individual specification TCK should include tests of all types of requirements and must contain tests of Type 2 requirements, if they exist, and of Type 3 requirements.
 
-The type 3 tests must be executed in compatible implementations of the profiles to validate the individual specification implementation functions properly in the profile implementation: platform, web profile or core profile.
+The tests of Type 2 or Type 3 requirements must be executed in compatible implementations of the profiles to validate the individual specification implementation functions properly in the profile implementation: platform, web profile or core profile.
+
+The individual specification TCK must provide a mechanism for running the required TCK tests (Types 1 and 3) outside the context of a platform or profile. Type 2 tests must be excluded and not run. This mechanism is to be used by standalone implementations of the specification who wish to certify their compatibility.
+
+A profile-ready mechanism must also be provided for running the required TCK tests (Types 2 and 3) in the profiles in which they are included: platform, web profile or core profile. Type 1 tests must be excluded and not run. This mechanism is to be used by platform or profile implementations who wish to certify their compatibility with the specification.
+
+The requirement to produce a profile-ready mechanism does not mean a platform or profile implementation must pass the individual specification TCK using this mechanism before the individual specification can perform a release. If problems with the tests when run using the profile-ready mechanism are found after a specification release, they can be addressed via the TCK challenge mechanism or by a new release of the individual specification.
```

Comment:
In the past, we haven't used the TCK challenge mechanism prior to the final Platform/profile release (that was discussed by the Specification Committee for EE 10 from what I recall).  I'm not against new releases of the Specification but I don't think that is needed to change a TCK prior to the Platform/Profile release.  

Could it make sense to update to:
`they can be addressed by an update of the Specification TCK.`  ?

Reply:
@scottmarlow 

I think what you suggest is fine, but before updating the PR I'll wait and see if Emily or others disagree.

Both my 'via the TCK challenge mechanism' and my 'by a new release of the individual specification' are processes that may lead to an update of the specification TCK. But this section of the document doesn't need to enumerate those processes; your simple general wording covers them all.

But, for consistency with the rest of the section I'd say 'update of the individual specification TCK', not 'Specification TCK'.

(If I get some time I'll double check whether my usage is consistent with the rest of the document.)

BTW this paragraph may veer from my aim of not changing the intended meaning of the existing content. Not because I wanted to but perhaps due to ignorance of the existing practices. I just wanted to account for the fact we discussed on the mail list that tests that move out of the platform TCK may have problems that aren't discovered until after the spec is released. So this is definitely a bit that needs careful attention.

Thanks!

Reply:
:+1:  for changing to ` update of the individual specification TCK' + also updating the version to 1.4.1 in the title as Emily mentioned.